docs/topic-guides/kotlin/suggest.md

```diff
@@ -0,0 +1,97 @@
+# Suggest
+
+The API for the `SuggestStore` can be found in the [MozillaComponents Kotlin documentation](https://mozilla.github.io/application-services/kotlin/kotlin-components-docs/mozilla.appservices.suggest/-suggest-store/index.html).
+
+> The `SuggestStore` is a synchronous, which needs to be wrapped in the asynchronous primitive of the target language you are using it in.
+
+## Setting up the store
+
+You need to import the following primitives to work with the `SuggestStore` (these come from the generated `suggest.swift` file, produced by `uniffi`):
+
+```kotlin
+import mozilla.appservices.remotesettings.RemoteSettingsServer
+import mozilla.appservices.suggest.SuggestApiException
+import mozilla.appservices.suggest.SuggestIngestionConstraints
+import mozilla.appservices.suggest.SuggestStore
+import mozilla.appservices.suggest.SuggestStoreBuilder
+import mozilla.appservices.suggest.Suggestion
+import mozilla.appservices.suggest.SuggestionQuery
+```
+
+On start up of your application, you create a `SuggestStore` (as a singleton). You do this via the `SuggestStoreBuilder`, which returns a `SuggestStore`:
+
+```kotlin
+internal val store: Lazy<SuggestStore> = lazy {
+    SuggestStoreBuilder()
+        .dataPath(context.getDatabasePath(DATABASE_NAME).absolutePath)
+        .remoteSettingsServer(remoteSettingsServer)
+        .build()
+```
+
+In addition to the `dataPath`, which is the path (the SQLite location) where you store your suggestions, you need a `remoteSettingsServer`, which is the URL to the instance of `RemoteSettings` you will be using (e.g. `https://remote-settings-dev.allizom.org`)
```

Comment:
The `remoteSettingsServer` is only needed if you want to use something other than `prod`.  They pass in a `RemoteSettingsServer` object rather than a raw URL (https://mozilla.github.io/application-services/kotlin/kotlin-components-docs/mozilla.appservices.remotesettings/-remote-settings-server/index.html).

Reply:
> The remoteSettingsServer is only needed if you want to use something other than prod.

I don't quite understand. I assumed we always pass the object, just with a different Url?

Reply:
I think it's optional and you could skip the `.remoteSettingsServer(remoteSettingsServer)` call.  In practice though, consumers will probably always pass it so maybe it's not worth mentioning.